templates/templates/cart.html

```diff
@@ -0,0 +1,152 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:group {"layout":{"inherit":true,"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:woocommerce/cart -->
+	<div class="wp-block-woocommerce-cart alignwide is-loading"><!-- wp:woocommerce/filled-cart-block {"align":"wide"} -->
+		<div class="wp-block-woocommerce-filled-cart-block alignwide"><!-- wp:woocommerce/cart-items-block -->
+			<div class="wp-block-woocommerce-cart-items-block"><!-- wp:woocommerce/cart-line-items-block -->
+				<div class="wp-block-woocommerce-cart-line-items-block"></div>
+				<!-- /wp:woocommerce/cart-line-items-block -->
+
+				<!-- wp:woocommerce/cart-cross-sells-block -->
+				<div class="wp-block-woocommerce-cart-cross-sells-block"><!-- wp:heading {"fontSize":"large"} -->
+					<h2 class="wp-block-heading has-large-font-size">You may be interested in…</h2>
```

Comment:
What happens with translations here?

Reply:
Well caught. Once more, this comes to mind p1685013363144909-slack-C8X6Q7XQU

What do you think about it @mikejolley ?

Reply:
Typically this is handled by putting this into a block pattern (which can be [either dynamically loaded via PHP file](https://github.com/woocommerce/woocommerce-blocks/tree/trunk/patterns) - which is automatically handled by our loader system), or dynamically registered via PHP to include localized strings. Then you just put the pattern comment in the `.html` template file. You'll just want to make sure the pattern is configured to not show up in the inserter.

Reply:
[Issue](https://github.com/woocommerce/woocommerce-blocks/issues/9865) created.